cmake/Dependencies.cmake

```diff
@@ -28,55 +28,8 @@ endif()
 # For tox-bootstrapd.
 pkg_search_module(LIBCONFIG   libconfig IMPORTED_TARGET)
 
-###############################################################################
-#
-# :: For MSVC Windows builds.
-#
-# These require specific installation paths of dependencies:
-# - libsodium in third-party/libsodium/Win32/Release/v140/dynamic
-# - pthreads in third-party/pthreads-win32/Pre-built.2
-#
-###############################################################################
-
 if(MSVC)
-  # libsodium
-  # ---------
-  if(NOT LIBSODIUM_FOUND)
-    find_library(LIBSODIUM_LIBRARIES
-      NAMES sodium libsodium
-      PATHS
-        "third_party/libsodium/Win32/Release/v140/dynamic"
-        "third_party/libsodium/x64/Release/v140/dynamic"
-    )
-    if(LIBSODIUM_LIBRARIES)
-      include_directories("third_party/libsodium/include")
-      set(LIBSODIUM_FOUND TRUE)
-      message("libsodium: ${LIBSODIUM_LIBRARIES}")
-    else()
-      message(FATAL_ERROR "libsodium libraries not found")
-    endif()
-  endif()
-
-  # pthreads
-  # --------
-  if(CMAKE_USE_WIN32_THREADS_INIT)
-    find_library(CMAKE_THREAD_LIBS_INIT
-      NAMES pthreadVC2
-      PATHS
-        "third_party/pthreads-win32/Pre-built.2/lib/x86"
-        "third_party/pthreads-win32/Pre-built.2/lib/x64"
-    )
-    if(CMAKE_THREAD_LIBS_INIT)
-      include_directories("third_party/pthreads-win32/Pre-built.2/include")
-      add_definitions(-DHAVE_STRUCT_TIMESPEC)
-      message("libpthreads: ${CMAKE_THREAD_LIBS_INIT}")
-    else()
-      find_package(pthreads4w)
-      if(NOT pthreads4w_FOUND)
-        message(FATAL_ERROR "libpthreads libraries not found")
-      endif()
-      include_directories(${pthreads4w_INCLUDE_DIR})
-      link_libraries(${pthreads4w_LIBRARIES})
-    endif()
-  endif()
+  # For toxcore.
+  find_package(PThreads4W REQUIRED)
```

Comment:
vcpkg internally aliases "pthreads" to "PThreads4W" on windows.
I use code like this:
```cmake
          find_package(pthreads QUIET)
          if(TARGET PThreads4W::PThreads4W)
              target_link_libraries(toxcore PThreads4W::PThreads4W)
          else()
              set(THREADS_PREFER_PTHREAD_FLAG ON)
              find_package(Threads REQUIRED)
              target_link_libraries(toxcore Threads::Threads)
          endif()
```
not sure if we want that

Reply:
Done.